src/network/networkd-network-gperf.gperf

```diff
@@ -109,7 +109,7 @@ Network.DNSSEC,                              config_parse_dnssec_mode,
 Network.DNSSECNegativeTrustAnchors,          config_parse_dnssec_negative_trust_anchors,               0,                             0
 Network.NTP,                                 config_parse_ntp,                                         0,                             offsetof(Network, ntp)
 Network.IPForward,                           config_parse_address_family_with_kernel,                  0,                             offsetof(Network, ip_forward)
-Network.IPMasquerade,                        config_parse_bool,                                        0,                             offsetof(Network, ip_masquerade)
+Network.IPMasquerade,                        config_parse_address_family_with_kernel,                  0,                             offsetof(Network, ip_masquerade)
```

Comment:
do not add new uses of config_parse_address_family_with_kernel() please. use config_parse_address_family() instead for new stuff

Reply:
I did not find a suitable function; I've added config_parse_address_family() now.